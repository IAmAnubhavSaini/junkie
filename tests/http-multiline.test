#!/bin/sh

NAME="HTTP-multiline"
PCAPS="pcap/http/http_multiline.pcap"
filter() {
	# We keep only the GET because:
	# 1- it's enough to show that chunks are decoded properly
	# 2- proto for info are likely to change in the future (now dumper works as a per-packet basis which is not perfect)
	grep -e '^HTTP.*method=GET' -e '^[[:space:]]'
}
OUTPUT="http-multiline.out"

. ./dump_test
