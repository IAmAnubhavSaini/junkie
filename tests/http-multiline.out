-- pcap/http/http_multiline.pcap --
HTTP: head_len=155, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.com, user_agent=curl/7.24.0 (x86_64-pc-linux-gnu) libcurl/7.24.0 OpenSSL/1.0.0j zlib/1.2.5.1, referrer=unset, url=/no-such-page, pkts=1
HTTP: head_len=159, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=
			 www.google.com, user_agent=curl/7.24.0 (x86_64-pc-linux-gnu) libcurl/7.24.0 OpenSSL/1.0.0j zlib/1.2.5.1, referrer=unset, url=/no-such-page, pkts=1
HTTP: head_len=164, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=
 
 
 
 
 www.google.com, user_agent=curl/7.24.0 (x86_64-pc-linux-gnu) libcurl/7.24.0 OpenSSL/1.0.0j zlib/1.2.5.1, referrer=unset, url=/no-such-page, pkts=1
HTTP: head_len=163, payload=0, method=GET, code=unset, content_length=unset, transfert_encoding=unset, mime_type=unset, host=www.google.com, user_agent=
 curl/7.24.0
 (x86_64-pc-linux-gnu)
 libcurl/7.24.0
 OpenSSL/1.0.0j
 zlib/1.2.5.1
  , referrer=unset, url=/no-such-page, pkts=1
HTTP: head_len=86, payload=0, method=GET, code=unset, content_length=0, transfert_encoding=unset, mime_type=unset, host=unset, user_agent=unset, referrer=unset, url=/no-such-page, pkts=1
